app/helpers/application_helper.rb

```diff
@@ -1,2 +1,6 @@
 module ApplicationHelper
+  def full_title page_title = ""
+		base_title = "Ruby on Rails Tutorial Sample App"
```

Comment:
check lại toàn bộ indent. indent 2 space nhé
e tìm hiểu và áp dụng i18n cho toàn bộ text bao gồm cả ở view nhé
gem i18n